jobs/dap-collector-ppa-dev/dap_collector_ppa_dev/main.py

```diff
@@ -154,6 +165,23 @@ async def collect_once(task, timestamp, duration, hpke_private_key, auth_token):
     return res
 
 
+def parse_histogram(histogram_str):
```

Comment:
The method name is confusing given the comment that we're using vectors and not histograms currently. Does this name make sense given the response shape of the collect call?

Reply:
Good call, renamed it to `parse_vector`.